### PR TITLE
AzureRmWebAppDeploymentV3 upgraded to Node16

### PR DESCRIPTION
**Pull request by ismayilov-ismayil**
**Task name**:  AzureRmWebAppDeploymentV3 upgraded to Node16

**Description**: Added Node 16 handler

**Documentation changes required:** N
**Added unit tests:** N

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

### CODE REVIEW

**Comment by vmapetr** [comment]
There is no execution handler for Node16 - is this expected?

**Review by LiliaSabitova — APPROVED** [review]

**Review by vmapetr — APPROVED** [review]

File: Tasks/AzureRmWebAppDeploymentV3/task.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
     "version": {
         "Major": 3,
         "Minor": 215,
-        "Patch": 0
+        "Patch": 1
```
Comment (KonstantinTyukalov) [inline Tasks/AzureRmWebAppDeploymentV3/task.json:21]: I think it should now be 216.0

**Review by KonstantinTyukalov — APPROVED** [review]

**Review by kirill-ivlev — APPROVED** [review]

**Comment by ozanovus** [comment]
Hello, this change is failing, could you please take a look?

image

**Pull request merged**